2021-03-18-if-else-exercises/main.js

```diff
@@ -0,0 +1,61 @@
+/* 
+    Exercise 1:
+    - complete the if/else statement so that the result is true if the input number is bigger equal than 70 and smaller equal than 99 
+    - try changing the input and check if it's still correct
+*/
+
+let inputEx1 = 5;
+
+if () {
+    console.log("#1 Is " + inputEx1 + "between 70 and 99?", true);
+} else {
+    console.log("#1 Is " + inputEx1 + "between 70 and 99?", false);
+}
+
+/*
+    Exercise 2:
+    - given 2 numbers in inputs, write an if/else statement that prints true if at least one of them is in the range between 70 and 900, false otherwise
```

Comment:
```suggestion
    - given 2 numbers in inputs, write an if...else statement that prints true if at least one of them is in the range between 70 and 900, false otherwise
```